lib/auth/profile.js

```diff
@@ -9,16 +9,16 @@ module.exports = (endpoint) => {
 
   const request = api(endpoint);
 
-  return (token, session) => {
+  return (user, session) => {
 
     return Promise.resolve()
       .then(() => {
-        if (!session.profile ||
-          (session.profile &&
-            moment().isAfter(moment(session.profile.expiresAt)))) {
+        if (session.profile && Date.now() < session.profile.expiresAt && session.profile.userId && session.profile.userId === user.id) {
```

Comment:
I can't find any linting enabled for line length - any pref?

Reply:
Correct.

:trollface:

Reply:
Good job I've got a 21:9 monitor then :D

Reply:
Updated.

Reply:
Agreed, I'd go further and say avoid abbreviations in variable names as well.

My issue with long lines is that I tend to use side-by-side diffs, and lines over about 120 char auto-wrap and make them more difficult to scan read.